### PR TITLE
Add BN to deep torch and fix a small Warning statement

### PR DESCRIPTION
**Pull request by JiechengZhao**
Add BatchNorm layer to the deep torch. When the eval is open it is simply a linear layer.

Fix a bug in a warning statement.

### CODE REVIEW

**Comment by slundberg** [comment]
Thanks! I think it's safe to assume that people are not explaining a batch norm that is not fixed. Otherwise we would need to explicitly check for that.
